src/api/app/views/webui/user/show.html.haml

```diff
@@ -12,6 +12,10 @@
       %br/
     %span#home-username= @displayed_user.login if @displayed_user.login
   / mail
+  %p
+    = label_tag 'cloud_backend_upload_job[vpc_subnet_id]', 'VPC Subnet ID (eg. subnet-12345678, optional):'
```

Comment:
Huh? Why on the user show page? I guess this is a leftover from testing?

Reply:
yes